package-system/AWSGameLiftServerSDK/docker_build_linux.sh

```diff
@@ -27,35 +55,64 @@ build_package() {
 
     echo "Generating $1"
 
-    cmake -G Ninja -S ${SRC_PATH} -B $1 -DBUILD_SHARED_LIBS=$2 -DCMAKE_BUILD_TYPE=Release
+    lib_type=$1
+    if [ "$lib_type" == "Shared" ]
+    then
+        build_shared=ON
+    else
+        build_shared=OFF
+    fi
+
+    echo cmake -S ${SRC_PATH} -B ${BUILD_PATH_ROOT}/${lib_type} \
+          -G "Unix Makefiles" \
+          -DBUILD_SHARED_LIBS=$build_shared \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_MODULE_PATH=$DOWNLOADED_PACKAGE_FOLDERS
+
+
+    cmake -S ${SRC_PATH} -B ${BUILD_PATH_ROOT}/${lib_type} \
+          -G "Unix Makefiles" \
+          -DBUILD_SHARED_LIBS=$build_shared \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_MODULE_PATH=$DOWNLOADED_PACKAGE_FOLDERS
     if [ $? -ne 0 ]
     then
-        echo "Error generating AWS Gamelift Server SDK for $1" 
+        echo "Error generating AWS Gamelift Server SDK build" 
         exit 1
-    fi
-    cmake --build $1
+    fi          
+
+    echo "cmake --build ${BUILD_PATH_ROOT}/${lib_type}"
+    cmake --build ${BUILD_PATH_ROOT}/${lib_type}
     if [ $? -ne 0 ]
     then
-        echo "Error building AWS Gamelift Server SDK for $1" 
+        echo "Error building the ${lib_type}"
         exit 1
-    fi
+    fi          
```

Comment:
Trailing whitespace has been added to this line.
```suggestion
    fi
```